admin_guide/install/install_defender/install_serverless_defender_autoprotect.adoc

```diff
@@ -10,6 +10,9 @@ Serverless auto-protect supports the following runtimes:
 * Node.js 10.x
 * Python 2.7, 3.6, and 3.7
 
+=== Limitations
+
+* AWS Lambda has a limit of xref:https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-limits.html[5 layers] per function. If you plan to use mutiple layers and might reach this limit, consider choosing the xref:install_serverless_defender.adoc[embedded] option.
```

Comment:
@avish9 What happens if you've got a lot of Lambda functions in scope for auto-protect, and most of them have fewer than 5 layers, but some have more than 5 layers? Do you get some type of error response that enumerates all functions that weren't protected? Where can you review the error response?

Reply:
You'll see a general error in the autoprotect screen and you'll have more information in the console's logs. Unfortunately, we didn't make it to adding a screen that shows per lambda protection status here, but a user can see which functions are protected in the serverless radar.

Reply:
Got it, thanks @avish9